src/components/user/tour/utils/constants.js

```diff
@@ -0,0 +1,50 @@
+export const TourColumns = [
```

Comment:
Use the right naming convention. 

Variables: `tourColumns`
Constants: `tourColumns` or `TOUR_COLUMNS`
Constructor functions or class names: `TourColumns`